### PR TITLE
New WebKit tests targetting fetch API stream support

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

**Comment by jdm** [comment]
Are we good to merge here, or should someone else take a look at these tests first?

File: fetch/api/resources/utils.js [hunk 1]
```diff
@@ -72,3 +72,11 @@ function validateStreamFromString(reader, expectedValue, retrievedArrayBuffer) {
     validateBufferFromString(retrievedArrayBuffer, expectedValue, "Retrieve and verify stream");
   });
 }
+
+// From streams tests
+function delay(milliseconds)
```
Comment (domenic) [inline fetch/api/resources/utils.js]: The rest of this file uses two-space indents and (with one exception) no newline before `{`.

**Comment by domenic** [comment]
These tests LGTM besides a nit I noted above that doesn't really matter. Mostly wanted to ping @tyoshino and @yutakahirano so they know we should import these to Chrome.

**Pull request merged**